samples/tab-deeplink/nodejs/pages/ExtendedDeepLink.html

```diff
@@ -0,0 +1,217 @@
+                if (people) {
+                    document.getElementById("Attendee").value = (people.map((p) => p.email)).join(",");
+                }
+            }).catch((error) => {
+                console.log(error);
+            });
+        });
+    }
+
+    function onCallDeepLinkButtonClick(callModalities) {
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+                if (people) {
+                    if (microsoftTeams.call.isSupported()) {
+                        microsoftTeams.call.startCall({
+                            targets: people.map((p) => p.email),
+                            requestedModalities: [callModalities]
+                        })
+                            .then((result) => {
+                                console.log(result);
+                            }).catch((error) => {
+                                console.log(error);
+                            });
+                    }
+                    else {
+                        alert("Call isn't supported");
+                    }
+                }
+            }).catch((error) => {
+                console.log(error);
+            });
+        });
+    }
+
+    function onSchedulingDialogClick() {
+        var subjectInput = document.getElementById("Subject");
+        var startTimeInput = document.getElementById("StartTime");
+        var endTimeInput = document.getElementById("EndTime");
+        var contentInput = document.getElementById("Content");
+        var attendeeInput = document.getElementById("Attendee");
+
+        if (subjectInput.value.trim() !== "" && startTimeInput.value !== "" && endTimeInput.value !== "" && attendeeInput.value !== "") {
+            var meetingDetails =
+            {
+                attendees: attendeeInput.value,
+                content: contentInput.value.trim(),
+                endTime: endTimeInput.value,
+                startTime: startTimeInput.value,
+                subject: subjectInput.value.trim(),
+            };
+
+            // Currently Calendar API are not supported in Microsot Teams
+            // microsoftTeams.app.initialize().then(() => {
+            //     microsoftTeams.calendar.composeMeeting(meetingDetails).
+            //         then((result) => { console.log(result); }).
+            //         catch((error) => { console.log(error); });
+            // });
+
+            microsoftTeams.app.openLink(`https://teams.microsoft.com/l/meeting/new?subject=${meetingDetails.subject}&startTime=${meetingDetails.startTime}&endTime=${meetingDetails.endTime}&content=${meetingDetails.content}&attendees=${meetingDetails.attendees}`)
+        }
+    }
+
+    function onAppInstallDialogClick() {
+        microsoftTeams.app.openLink("https://teams.microsoft.com/l/app/1542629c-01b3-4a6d-8f76-1938b779e48d");
```

Comment:
Should we hard code the app id or add placeholder and add note in readme?

Reply:
It is just to show how it works so this would be enough. Suggested by Wajeed only.